src/cmd/ksh93/sh/path.c

```diff
@@ -1057,24 +1057,12 @@ pid_t path_spawn(Shell_t *shp, const char *opath, char **argv, char **envp, Path
     sfsync(sfstderr);
     sh_sigcheck(shp);
     path = path_relative(shp, opath);
-    if (*path != '/' && path != opath) {
```

Comment:
This change has been already merged into `master`. This commit should not be in this pull request.

Reply:
Correct. I accidentally opened the PR before pushing to master the branch this was based on. It's harmless.